### ISSUE TITLE
[OAKTrust] All Pages

### ISSUE THREAD

**Issue by devangm** [TAMULib/dspace-angular#201]
**URL:** https://oaktrust-dev.library.tamu.edu/home
**WCAG 2.1 AA compliant? :** No
**Violation:** 2.1.1 Keyboard (A)
**Issues:** Can't navigate Browse OAKTrust dropdown with keyboard
**Notes:** Can't navigate to other options in the dropdown menu, nor can you select any element inside "All of OAKTrust"
